tests/dummy/public/test-forms/tables/form.ttl

```diff
@@ -38,6 +38,7 @@ ext:administrativeUnitsL a form:Listing;
   form:addLabel "Voeg nieuw betrokken lokaal bestuur toe";
   form:canRemove true;
   form:removeLabel "Verwijder";
+  form:showTableRowIndex true;
```

Comment:
We could also add something like 

```ttl
form:tableRowIndexPattern "Stap {}";
```

Where we could then replace the `{}` (or whatever placeholder pattern we choose), to display something like `Stap 1` to the user. But that might be very niche and not worth the effort?

Reply:
Since the column header can also be configured now which covers the same use case.